client/my-sites/stats/activity-log-banner/progress-banner.jsx

```diff
@@ -10,13 +10,17 @@ import { localize } from 'i18n-calypso';
 import ActivityLogBanner from './index';
 import ProgressBar from 'components/progress-bar';
 
-function SuccessBanner( {
+function ProgressBanner( {
 	moment,
 	percent,
 	status,
 	timestamp,
 	translate,
 } ) {
+	const restoreStatusDescription = status === 'queued'
+		? translate( 'Hold tight, your restore will begin soon.' )
+		: translate( 'Sit back and relax, your site is currently being restored.' );
```

Comment:
I love the addition of 'holding' text while waiting for the restore to begin. I do have concerns with the tone. If I'm in the middle of a crisis with a site, I probably want to see messages that are as plain and to-the-point as possible.